cos_alerter/daemon.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import datetime
+import durationpy
+import json
+import signal
+import subprocess
+import sys
+import textwrap
+import time
+
+from .alerter import DataWriter, config, send_notifications
+
+
+def notify(time_data):
+    if not time.time() - time_data.notify_time > durationpy.from_str(config['notify']['repeat_interval']).total_seconds():
+        #TODO log something here
+        return
+    time_data.notify_time = time.time()
+    last_alert_time = datetime.datetime.fromtimestamp(time_data.alert_time).strftime('%A, %B %d %Y %I:%M%p %Z')
```

Comment:
If we're gonna format, may as well use ISO8601 for the format.

Reply:
I personally think "Thursday, March 23 2023 11:01AM" looks a lot better than "2023-03-23T11:01:16.136164"

Reply:
Done.